.github/workflows/benchmarks.yml

```
@@ -51,11 +51,13 @@ jobs:
           ${{ runner.os }}-pip-
 
     - name: Install ubuntu dependencies
+      env:
+        DEBIAN_FRONTEND=noninteractive
```

```suggestion
        DEBIAN_FRONTEND: noninteractive
```

Not sure why this is needed? But i think this should be like ^

---

Oh, just saw the chat in IPS channel. Sounds like we could just drop it anyway?

---

Thanks - is this how we set env variables?

---

Yeah, the Benchmark workflow didn't even run before. Obviously because of wrong syntax.